src/main/resources/mixins.common.command.json

```diff
@@ -0,0 +1,17 @@
+{
+    "minVersion": "0.5.1",
+    "package": "org.spongepowered.common.mixin.command",
+    "refmap": "mixins.common.refmap.json",
+    "plugin": "org.spongepowered.common.mixin.plugin.CommandPlugin",
```

Comment:
This plugin name (and the mixin config name) should reflect that this is for "multi world command patches", not commands in general.